src/main/scala/org/ergoplatform/nodeView/history/ErgoSyncInfo.scala

```diff
@@ -13,14 +13,14 @@ import scorex.util.{ModifierId, ScorexLogging, bytesToId, idToBytes}
   *
   */
 sealed trait ErgoSyncInfo extends SyncInfo {
-  /*
+  /**
    * Whether sync info message corresponds to non-empty blockchain
    */
   val nonEmpty: Boolean
 
   override type M = ErgoSyncInfo
 
-  override lazy val serializer: ScorexSerializer[ErgoSyncInfo] = ErgoSyncInfoSerializer
+  override lazy val serializer: ErgoSerializer[ErgoSyncInfo] = ErgoSyncInfoSerializer
```

Comment:
ScalaDoc for `override` method is not required as long as the base class is documented. Please document BytesSerializable.

Reply:
ScalaDoc for the method is still missing.

Reply:
added